### ISSUE TITLE
[GalacticScale 2.1.9] Unable to land on a planet

### ISSUE THREAD

**Issue by phoenixx-666** [Touhma/DSP_Plugins#138]
Unable to land on a planet **Kusomonmon - XV - I** (an Ice planet satellite of an Ice Giant) - Mecha just slips through it even at a small speed.
The cluster is the same as in [this issue](https://github.com/Touhma/DSP_Plugins/issues/137).

**Comment by Meallia** [comment]
Same issue here, cannot land on a moon. It seems like its too close to its Ice Giant, when I'm about to land my minimap indicates that I'm on the ice giant and the moon surfaces does not load.

I attached my config and the save file right before landing.

[Preferences.zip](https://github.com/Touhma/DSP_Plugins/files/7072796/Preferences.zip)
[Planet won't load - before landing.zip](https://github.com/Touhma/DSP_Plugins/files/7072795/Planet.won.t.load.-.before.landing.zip)

**Comment by innominata** [comment]
Should be fixed in 2.1.13 :)

**Comment by nclow** [comment]
Encountering the same issue in 2.2.0.17 with DSP 0.8.23.9989. Any body that is the satellite of a gas or ice giant simply cannot be landed on. I tried to land on half a dozen of these types of moons all over the cluster and simply fly right through them at any speed. The game seed was 72736089-64-A10, although when I put this seed in again to regenerate I got a different cluster configuration with different planet names for some reason.
